sled-agent/src/rack_setup/service.rs

```diff
@@ -16,31 +16,30 @@
 //! Rack setup occurs in distinct phases that are denoted by the presence of
 //! state files that get generated as RSS executes:
 //!
-//! - /pool/int/UUID/config/rss-sled-plan.json (Sled Plan)
-//! - /pool/int/UUID/config/rss-service-plan-v5.json (Service Plan)
-//! - /pool/int/UUID/config/rss-plan-completed.marker (Plan Execution Complete)
+//! - /pool/int/UUID/config/rss-started.marker (RSS has started)
+//! - /pool/int/UUID/config/rss-plan-completed.marker (RSS Complete)
 //!
-//! These phases are described below.  As each phase completes, a corresponding
-//! state file is written.  This mechanism is designed so that if RSS restarts
-//! (e.g., after a crash) then it will resume execution using the same plans.
+//! In its current incarnation, RSS is not capable of restarting without a
+//! clean-slate of the rack. Rather than persisting plan files which may let
+//! us pick up and restart at certain given points we instead only record two
+//! marker files. These marker files indicate one of two things:
 //!
-//! The service plan file has "-v2" in the filename because its structure
-//! changed in omicron#4466.  It is possible that on startup, RSS finds an
-//! older-form service plan.  In that case, it fails altogether.  We do not
-//! expect this condition to happen in practice.  See the implementation for
-//! details.
+//!   * RSS has run already but failed to complete. We must clean-slate the
+//!     rack if we try to run RSS again and see this file.
```

Comment:
The marker also exists on _successful_ RSS runs, right (i.e., nothing removes it)? Maybe reword this to something like

```suggestion
//!   * RSS has started. We must clean-slate the rack if we try to start
//!      RSS and see this file, as it indicates the original RSS attempt failed.
```

Reply:
Good call. Done in [ddb7cc3](https://github.com/oxidecomputer/omicron/pull/7185/commits/ddb7cc324ca822f24e35587051cc4e78b95c7b47)